internal/consensus/metrics.go

```diff
@@ -212,12 +235,28 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Name:      "block_parts",
 			Help:      "Number of blockparts transmitted by peer.",
 		}, append(labels, "peer_id")).With(labelsAndValues...),
-		StepTime: prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
+		BlockGossipReceiveTime: prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "block_gossip_receive_time",
```

Comment:
Consider possibly `_latency` instead of `_time` to emphasize that it's not a sample of moments but a sample of durations?